examples/audit-log-example.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+error_reporting(E_ALL);
+
+/**
+ * This file contains examples of Smartling API 2.x usage.
+ *
+ * How to use:
+ * run "php example.php --project-id={PROJECT_ID} --user-id={USER_IDENTIFIER} --secret-key={SECRET_KEY}" in console
+ *
+ * Be sure you that dependencies are solved bu composer BEFORE running.
+ */
+
+$longOpts = [
+    'project-id:',
+    'user-id:',
+    'secret-key:',
+    'account-uid:'
+];
+
+$options = getopt('', $longOpts);
+
+if (
+    !array_key_exists('project-id', $options)
+    || !array_key_exists('user-id', $options)
+    || !array_key_exists('secret-key', $options)
+) {
+    echo 'Missing required params.' . PHP_EOL;
+    exit;
+}
+
+$autoloader = '../vendor/autoload.php';
+
+if (!file_exists($autoloader) || !is_readable($autoloader)) {
+    echo 'Error. Autoloader not found. Seems you didn\'t run:' . PHP_EOL . '    composer update' . PHP_EOL;
+    exit;
+}
+else {
+    /** @noinspection UntrustedInclusionInspection */
+    require_once '../vendor/autoload.php';
+}
+
+$projectId = $options['project-id'];
+$accountUid = $options['account-uid'];
+$userIdentifier = $options['user-id'];
+$userSecretKey = $options['secret-key'];
+$authProvider = \Smartling\AuthApi\AuthTokenProvider::create($userIdentifier, $userSecretKey);
+
+/**
+ * @param \Smartling\AuthApi\AuthApiInterface $authProvider
+ * @param string $projectId
+ * @return bool
+ */
+function searchProjectLevelLogRecordDemo($authProvider, $projectId)
+{
+    $response = false;
+    $auditLog = \Smartling\AuditLog\AuditLogApi::create($authProvider, $projectId);
+    $st = microtime(true);
+
+    try {
+        $searchParams = (new \Smartling\AuditLog\Params\SearchRecordParameters())
+            ->setSearchQuery('foo:bar')
+            ->setOffset(0)
+            ->setLimit(100)
+            ->setSort('_id', \Smartling\AuditLog\Params\SearchRecordParameters::ORDER_DESC);
```

Comment:
Let's do example more real life and use `time` field.

Reply:
You mean to use `time` in `setSearchQuery()` method?

Reply:
Done